src/clients/db/PatientRepository.ts

```diff
@@ -1,8 +1,10 @@
 import escapeStringRegexp from 'escape-string-regexp'
```

Comment:
We should add tests in the `PatientRepository.test.tsx` to validate that this is working properly.

Reply:
I have added some tests.